examples/c/ksyscall.bpf.c

```diff
@@ -0,0 +1,28 @@
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+
+#define TASK_COMM_LEN 16
+
+SEC("ksyscall/kill")
+int BPF_KSYSCALL(entry_probe, pid_t tpid, int sig) 
+{
+	char comm[TASK_COMM_LEN] = {};
```

Comment:
there is no need to `= {}` initialize, `bpf_get_current_comm()` will ensure it's initialized